platform/core.network/src/org/netbeans/core/network/proxy/NetworkProxyReloader.java

```diff
@@ -78,8 +78,9 @@ public static void reloadNetworkProxy() {
         } else {
             LOGGER.log(Level.INFO, "System network proxy reloading succeeded."); //NOI18N
         }
-                       
-        switch (networkProxySettings.getProxyMode()) {
+
+        NetworkProxySettings.ProxyMode mode = networkProxySettings.getProxyMode();
+        switch (mode) {
             case AUTO:
```

Comment:
What is the purpose of this change?

Reply:
With this change it is easier for me to mangle the value of `mode` in debugger before the `switch` statement is executed.